src/pages/HomeParticipant.vue

```diff
@@ -186,14 +186,15 @@ const {
 });
 
 async function checkConsent() {
-  showConsent.value = false;
+  if (isLevante) return;
```

Comment:
This showConsent.value = false is needed because when switching administrations it needs to reset the value of showConsent
```suggestion
  showConsent.value = false
  
  if (isLevante) return;
```